reference/replay.tracker.events.md

```diff
@@ -149,21 +161,75 @@ Known values:
 
 - CampOwnershipFlag
 - DiabloShrine - Diablo's trait respawn point? Not unique.
-- KingsCore
-- RegenGlobe
+- KingsCore - The core of each team
+- RegenGlobe - A pickable unit that grants a small health and mana amount
+- RegenGlobeNeutral - A pickable unit that grants a small health and mana amount, usually drops when fighting bosses, it's color is purple.
 - StormGameStartPathingBlocker - Gate blockers at the beginning of the game?
 - StormGameStartPathingBlockerDiagonal - Gate blockers at the beginning of the game?
+- TownCannonTowerDead
 - TownCannonTowerL2
+- TownCannonTowerL2Standalone
 - TownCannonTowerL3
 - TownCannonTowerL3Standalone
+- TownGateL215BLUR
 - TownGateL215BRUL
+- TownGateL2BLUR
+- TownGateL315BLUR
+- TownGateL315BRUL
+- TownGateL3BRUL
 - TownMoonwellL2
 - TownMoonwellL3
 - TownTownHallL2
 - TownTownHallL3
+- TownWallRadial14L3
+- TownWallRadial15L3
+- TownWallRadial16L2
+- TownWallRadial17L2
+- TownWallRadial17L3
+- TownWallRadial18L2
+- TownWallRadial18L3
+- TownWallRadial19L1
+- TownWallRadial19L2
+- TownWallRadial19L3
+- TownWallRadial20L3
+- TownWallRadial21L3
+- TownWallRadial2L3
+- TownWallRadial3L3
 - TownWallRadial4L2
+- TownWallRadial4L3
 - TownWallRadial5L2
-- WeaponRackSpecialHeaven
+- TownWallRadial5L3
+- TownWallRadial6L2
+- TownWallRadial6L3
+- TownWallRadial7L2
+- TownWallRadial7L3
+- TownWallRadial8L3
+- TownWallRadial9L3
+- TownGateL2VerticalLeftVisionBlocked
+- TownGateL2VerticalRightVisionBlocked
+- TownGateL3BLURBRVisionBlocked
+- TownGateL3BLURTLVisionBlocked
+- TownGateL3BRULBLVisionBlocked
+- TownGateL3BRULTRVisionBlocked
+- TownGateL3VerticalLeftVisionBlocked
+- TownGateL3VerticalRightVisionBlocked
+- LuxoriaTemple - Temple you can controll in the Sky Temple Map
+- GhostShipBeacon - Ghost ship you can 'controll' by paying coins
+- ItemSoulPickup - Item you can pickup in the Tomb of the Spider Map, awards 1 gem.
+- ItemSoulPickupFive - Item you can pickup in the Tomb of the Spider Map, awards 5 gems.
+- SoulEater - Big spider summoned when a team completes the required amount of item souls.
+- SoulEaterMinion - Medium spiders summoned by the big spider.
+- ItemSoulPickupTwenty - Item you can pickup in the Tomb of the Spider Map, awards 20 gems.
+- MercLanerMeleeOgre - Siege mercenaries
+- MercLanerSiegeGiant - Boss mercenarie that appears on most of the maps
+- MercLanerRangedOgre - Another siege mercenarie
+- JunglePlantHorror
+- FootmanMinion - The normal minion
+- WizardMinion - Wizard minion, usually drops a health globe when it dies
+- RangedMinion - Ranged minion
+- CatapultMinion - The catapult
+- JungleGraveGolemDefender - Boss unit summoned in Cursed Mines map
+
```

Comment:
Hi Farof, can you please show me one example of how the formatting should be? I don't use a lot github's formatting markdown